src/work-allocation-2/containers/all-work-home/all-work-home.component.ts

```diff
@@ -40,6 +43,7 @@ export class AllWorkHomeComponent implements OnInit, OnDestroy {
 
     // Set up the page data.
     this.setupPageData(this.router.routerState.root.snapshot);
+    this.sessionStorageService.setItem(AllWorkHomeComponent.CLAIM_URL_RETURN_KEY, window.location.pathname);
```

Comment:
Why are you storing this in session Storage? Is that part of requirement?

Reply:
@udaydenduluri33 just thought it was the optimal solution to store where the user is coming from.

Reply:
@kuda1992 I dnt think it should be stored in session storage. This should be picked from router data and sent back. Its need not be persisted as its transient information. So, I suggest please remove it. Thanks.

Reply:
@kuda thanks for making the changes